### ISSUE TITLE
Error Declaration of CRM_Banking_PluginImpl_Importer_CODA with Civibanking 1.0-alpha2

### ISSUE THREAD

**Issue by hansrossel** [Project60/org.project60.coda#9]
with Civibanking 1.0-alpha2 there is a PHP Fatal error on /civicrm/banking/import:  Declaration of CRM_Banking_PluginImpl_Importer_CODA::getValue($key, $btx, $line = null, $header = []) must be compatible with CRM_Banking_PluginImpl_Importer_CSV::getValue($key, $btx, $line = null, $header = [], $params = []) in org.project60.coda/CRM/Banking/PluginImpl/Importer/CODA.php on line 207

I suppose the function should have $params: 
protected function getValue($key, $btx, $line = NULL, $header = array(), $params = []) {

**Comment by bjendres** [comment]
Yes, unfortunately the signature of the method being overwritten (``CRM_Banking_PluginImpl_Importer_CSV::getValue``) has changed at some point.

It should be sufficient to replace the line 207 you mentioned with
```
protected function getValue($key, $btx, $line=NULL, $header=[], $params = []) {
```

Let me know if that fixes it. If so, let me know and I'll create a release.

**Comment by hansrossel** [comment]
I can confirm the error on /civicrm/banking/import is fixed with this change.

**Comment by bjendres** [comment]
Hi @hansrossel, I created and attached a beta version, could you verify that it works, so I can finally release a ``1.0.0`` version?
